support-frontend/assets/helpers/checkouts.js

```diff
@@ -188,6 +193,21 @@ function getPaymentLabel(paymentMethod: PaymentMethod): string {
   }
 }
 
+// The value of result will either be:
+// . null - browser has no compatible payment method button)
+// . {applePay: true} - applePay is available
+// . {applePay: false} - GooglePay, Microsoft Pay and PaymentRequestApi available
+function getAvailablePaymentRequestButtonPaymentMethod(result: Object, contributionType: ContributionType):
```

Comment:
this is not particular to this PR but raising as a discussion point: should we talk within the stream about whether or not we use `function` or `const` declarations to write functions? I am used to doing them as `const`s but know this is a relatively new for JS. It doesn't matter in terms of code functionality, but it's nice to have consistent style across a codebase IMHO.